webapp/app/js/controllers/main.js

```diff
@@ -68,6 +68,18 @@ treeherder.controller('MainCtrl', [
                         thEvents.selectPreviousUnclassifiedFailure
                     );
 
+                } else if(ev.keyCode == 39) {
+                    // Highlight next job, key: right arrow
+                    $rootScope.$emit(
+                        thEvents.selectNextJob
+                    );
+
+                } else if(ev.keyCode == 37) {
+                    // Highlight previous job, key: left arrow
+                    $rootScope.$emit(
+                        thEvents.selectPreviousJob
+                    );
+
                 } else if (ev.keyCode === 32) {
```

Comment:
@dewgeek when you rebase to latest master before landing, you'll need to convert these to the new `Moustrap.bind` methods used in controllers/main in the repo. They may also require an additional digest cycle for each, depending whether you observe slowdowns in your local testing.